arangod/Aql/AqlItemBlockInputRange.h

```diff
@@ -57,6 +57,11 @@ class AqlItemBlockInputRange {
 
   std::pair<ExecutorState, InputAqlItemRow> peekDataRow() const;
 
+  /// @brief optimized version of peekDataRow, only to be used when it is known
+  /// that there is a next data row (i.e. if a previous call to hasDataRow()
+  /// returned true)
+  std::pair<ExecutorState, InputAqlItemRow> peekDataRow(HasDataRow) const;
```

Comment:
This doesn't seem to be used anywhere, or did I miss it?

Reply:
Should be used now.